calicoctl/calico_ctl/node.py

```diff
@@ -340,6 +342,9 @@ def _start_node_container_docker(ip, ip6, as_num, log_dir, node_image, detach, e
     calico_networking = os.getenv(CALICO_NETWORKING_ENV,
                                   CALICO_NETWORKING_DEFAULT)
 
+    calico_gobgp = os.getenv(CALICO_GOBGP_ENV,
+                             CALICO_GOBGP_DEFAULT)
+
```

Comment:
Added a commit to have a generic environment variable to select BGP implementation instead of adding CALICO_GOBGP.
I'm now naming it CALICO_NETWORKING_BACKEND tentatively.